six/two/two.cpp

```diff
@@ -830,7 +838,6 @@ void Two::setIsExcludedCb(cb_is_excluded_t cb)
 // Python Helpers
 
 // get_integration_list return a list of every datadog's wheels installed. The
```

Comment:
You forgot to remove the "The" at the end of the line which belonged to the sentence you removed.